bin/cli.js

```diff
@@ -149,18 +150,32 @@ const setRequestHeaders = () => {
   addUserAgentHeader('HubSpot CLI', pkg.version);
 };
 
+const shouldValidate = (options, validationList) => {
```

Comment:
Should I love this function to the `lib/validation.ts` file?

Reply:
You could if you want to. Or you could make the utils slightly more generic and keep them here. Really what this is doing is telling us whether the user-entered command matches a command that we've opted into a list. This could potentially have value in middleware outside of validation.

Like, what if this was something like `isTargetedCommand(options, commandMap)`, and it takes in a similar object for commandMap, but just more generic:

```json
const SKIP_ACCOUNT_VALIDATION = {
  accounts: {
    target: false,
    subCommands: {
      clean: { target: true },
      list: { target: true },
      remove: { target: true },
    },
  },
  sandbox: {
    target: false,
    subCommands: {
      delete: { target: true },
    },
  },
};
```

One other aspect of this is that we need to support command aliases. i.e. `hs sandboxes delete` should also skip validation. I'm not sure if there's a graceful way to do it. We might just need to duplicate the entries in the object.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1310/commits/dccf72fd6bd0bd7ea4b5a5b11374bc22179b0415. Making the `shouldTargetCommand` util recursive in another commit.